man/set-content-if-empty.md

```diff
@@ -0,0 +1,27 @@
+## set content if empty
+
+Creates a function that when passed to d3's `selection.select` operator will conditionally set the HTML content of a node (only if it is empty) and select the first child DOM element it finds.
```

Comment:
The "first child element" note has me confused, does this mean the template doesn't necessarily have to have a single root node? If the requirement is that the template has a single root node, then we can rephrase this as such:

> Creates a function that when passed to d3's `selection.select` operator will set the HTML content of a node *only if* it is *empty* and select the root node of the supplied template.